pyhf/tensor/dask_backend.py

```diff
@@ -0,0 +1,150 @@
+import dask.array as da
+import logging
+from scipy.special import gammaln
+from scipy.stats import norm
```

Comment:
hm not sure how standard `scipy` interfaces with non-evaluated arrays. somehow it works but not sure why

Reply:
it probably works because dask imlements generalized ufuncs .. 

http://dask.pydata.org/en/latest/array-gufunc.html